nominees/our-world-in-data.json

```diff
@@ -3,47 +3,134 @@
   "aliases": [
     "OWID"
   ],
-  "description": "Research and data to make and track progress against the world's largest problems. Allows for individuals and organisations to have a broad understanding of the state of the world. Includes tracking for SDGs.",
-  "website": "https://ourworldindata.org",
+  "description": "Research and data to make progress against the world’s largest problems. Allows for individuals and organisations to have a broad understanding of the state of the world. Includes tracking for SDGs.",
+  "website": "https://ourworldindata.org/",
   "license": [
     {
       "spdx": "CC-BY-4.0",
       "licenseURL": "https://ourworldindata.org/faqs#how-is-your-work-copyrighted"
+    },
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/owid/owid-grapher/blob/master/LICENSE.md"
     }
   ],
   "SDGs": [
```

Comment:
It seems that this could be a project that actually relates to every SDG. At first, I didn't realize that the URL provided went to a data portal created with/by OWID data.